app/controllers/issues_controller.rb

```diff
@@ -130,4 +141,11 @@ def issues_by_tags final
         issues.left_joins(:tags).where tags: { id: nil }
       )
     end
+
+    def survey_answer_params
+      params.require(:survey_answer).permit :survey_id,
+                                            answers_attributes: [:id, :type, 
+                                                                 :response_text, :drop_down_option_id,
+                                                                 :question_id]
```

Comment:
Esto lo modificaria a algo así
```suggestion
    def survey_answer_params
      params.require(:survey_answer).permit(
        :survey_id,
        answers_attributes: [:id, :type, :response_text, 
          :drop_down_option_id, :question_id
        ]
      )
```